### PR TITLE
Show blank square instead of grey man image for People without an image

### PR DESCRIPTION
**Pull request by tommyp**
This PR changes the `PersonPresenter` and view to display a white square if the Person has no image uploaded. [Ticket](https://trello.com/c/Fp39eiIQ/133-remove-stock-grey-man-image-from-across-gov-uk).

Commits:
- removing the blank person image
- return nil if the person has no image
- view and CSS changes
- remove/change some assertions in tests that look for the blank person image
- remove 2 undefined steps which were causing the test suite to fail
- remove the blank person image from the admin UI

Note the CSS for `.blank-person-inner:after` which has a defined percentage for `padding-top`. This is used to maintain an aspect ratio and make the empty div behave like an image.

Screenshot:
screen shot 2015-04-07 at 11 56 09

### CODE REVIEW

File: app/assets/stylesheets/frontend/helpers/_person.scss [hunk 1]
```diff
@@ -19,7 +19,35 @@
...
+    }
+    @include media(tablet) {
+      width: $full-width;
+      float: none;
+    }
+    .blank-person-inner {
+      display: block;
+      padding: 0 $gutter-half 0 0;
+      @include right-to-left {
+        padding: $gutter-one-third 0 0 $gutter-half;
```
Comment (edds) [inline app/assets/stylesheets/frontend/helpers/_person.scss]: Was adding this padding top on right-to-left intentional?

**Comment by edds** [comment]
One minor comment, otherwise this looks good to me.

File: app/assets/stylesheets/frontend/helpers/_person.scss (hunk 1 above)
Comment (tommyp) [inline app/assets/stylesheets/frontend/helpers/_person.scss]: Nope! Removed it and updated the commit.

**Comment by edds** [comment]
Looks good to merge to me.

**Pull request merged**